content/post/rxdnssd.md

```diff
@@ -364,8 +364,8 @@ mResolveSubscription = RxDNSSD.browse(mReqType, mDomain)
         });
 ~~~
 
-Creating of Rx chains is very usefull because you don't need to care about canceling all operations and removing listeners. The only thing you have to do inside your `onPause`||`onStop`||`onDestroy` methods is to call `mSubscription.cancel()`. Another big advantage is ability to handle all exceptions from chain in one place.
+Creating of Rx chains is very usefull because you don't need to care about canceling of all operations and listeners removal. The only thing you have to do inside your `onPause`||`onStop`||`onDestroy` methods is to call `mSubscription.cancel()`. Another big advantage is the ability to handle all exceptions from a chain in one place.
```

Comment:
"Another big advantage is the ability to.." from this sentence a reader might think that we've already discussed *this ability* to handle all exceptions (and this is some sort of a summary). Actually I don't remember anything about it in the article.. so it might be better to say "an ability". But screw it, it's not very important)